charts/mattermost-calls-offloader/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
```

Comment:
wondering if it make sense to change the `calls` helm chart to have a dependency on this chart. It's not for now just an idea to streamline the installation of the calls in k8s.

Reply:
This service enables functionalities which are still behind flags and default to off. In other words, the offloader isn't strictly needed for a working Calls deployment but only if someones wants to enable recordings (which require an enterprise license by the way). But again I am not sure what a dependency means in helm chart semantics.